module/documents/actor.mjs

```diff
@@ -136,23 +189,67 @@ export default class ActorEd extends Actor {
    */
    async rollEquipment( equipment, options = {} ) {
     const arbitraryStep = equipment.system.usableItem.arbitraryStep
+    let targets = game.users.current.targets
```

Comment:
see comments in `rollAbility`

Reply:
I added it to have it visible in the chat. Now it is visible which actor was targeted.